proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java

```diff
@@ -287,8 +287,13 @@ public ProctorResult determineTestGroups(
         for (final TestType testType : identifiers.getAvailableTestTypes()) {
             final String identifier = identifiers.getIdentifier(testType);
             if ((identifier != null) && !identifierValidator.validate(testType, identifier)) {
-                LOGGER.warn("An invalid identifier '" + identifier + "' for test type '" + testType + "'"
-                        + " was detected. Using fallback buckets for the test type.");
+                if (identifier.isEmpty()) {
+                    LOGGER.debug("An invalid identifier '" + identifier + "' for test type '" + testType + "'"
```

Comment:
Nit: should we extra "An invalid identifier '" + identifier + "' for test type '" + testType + "'"
                            + " was detected. Using fallback buckets for the test type." to a variable so it will not duplicate?

Reply:
Done